pose_lookup/plugin.cpp

```diff
@@ -27,6 +27,7 @@ class pose_lookup_impl : public pose_prediction {
         , align_trans{Eigen::Vector3f::Zero()}
         , align_quat{Eigen::Vector4f::Zero()}
         , align_scale{0.0} {
+        spdlogger(ILLIXR::getenv_or("POSE_LOOKUP_LOG_LEVEL", "off"));
```

Comment:
The expedient fix would be to just explicitly register the name "pose_lookup" here, similarly to how the main "illixr" plugin is currently used. Why doesn't pose_lookup from plugin or threadloop? Is this a unique case? Is it not really a plugin?

Reply:
We can also pass the logger object along when creating the service class from the plugin. This would save the trouble of creating a wrapper around the library.

Reply:
So something like - add a logger object as a member to pose_lookup_impl, then in pose_lookup_plugin's ctor, after calling pb->register_impl, set the logger?

Reply:
Yes, or we can put it into the constructor as well

Reply:
Okay, re-reading this and then with what was discussed at the 8/17 meeting, it turned out the most expedient fix was to remove the initialization the constructor and to change spdlog::get(name) to spdlog::get("illixr")